$NetBSD: patch-ar,v 1.1 2000/08/30 13:04:00 wennmach Exp $

Remove all "%" characters in front of register names to make our
m68k assembler happy.

--- lwp/process.m68k.S.orig	Wed Aug 30 14:46:46 2000
+++ lwp/process.m68k.S	Wed Aug 30 14:52:32 2000
@@ -75,19 +75,19 @@
 
 ENTRY(savecontext)
 	movb	#1,_C_LABEL(PRE_Block)	| Dont allow any interrupt finagling
-	link	%a6,#-(nregs*4)		| Save frame pointer & ...
+	link	a6,#-(nregs*4)		| Save frame pointer & ...
 					| ... allocate space for nregs registers
 /* Save registers */
-	moveml	#regs,%sp@
+	moveml	#regs,sp@
 
-	movl	%a6@(area1),%a0		| a0 = base of savearea
-	movl	%sp,%a0@(topstack)	| area->topstack = sp
-	movl	%a6@(newsp),%d0		| Get new sp
+	movl	a6@(area1),a0		| a0 = base of savearea
+	movl	sp,a0@(topstack)	| area->topstack = sp
+	movl	a6@(newsp),d0		| Get new sp
 	jeq	forw1			| If newsp == 0, no stack switch
-	movl	%d0,%sp			| Switch to new stack
+	movl	d0,sp			| Switch to new stack
 forw1:
-	movl	%a6@(f),%a0		| a0 = f
-	jbsr	%a0@			| f()
+	movl	a6@(f),a0		| a0 = f
+	jbsr	a0@			| f()
 
 /* It is impossible to be here, so abort() */
 
@@ -102,14 +102,14 @@
 area2	=	8
 
 ENTRY(returnto)
-	link	%a6,#0
-	movl	%a6@(area2),%a0		| Base of savearea
-	movl	%a0@(topstack),%sp	| Restore sp
+	link	a6,#0
+	movl	a6@(area2),a0		| Base of savearea
+	movl	a0@(topstack),sp	| Restore sp
 /* Restore registers */
-	moveml	%sp@,#regs
+	moveml	sp@,#regs
 
-	addl	#(nregs*4),%sp
-	movl	%sp,%a6			| Argghh...be careful here
-	unlk	%a6
+	addl	#(nregs*4),sp
+	movl	sp,a6			| Argghh...be careful here
+	unlk	a6
 	clrb	_C_LABEL(PRE_Block)
 	rts				| Return to previous process
